deeplabcut/pose_estimation_pytorch/apis/analyze_images.py

```diff
@@ -189,6 +191,7 @@ def superanimal_analyze_images(
         max_individuals=max_individuals,
         device=device,
         progress_bar=progress_bar,
+        # TODO: when COND_TOP_DOWN SuperAnimal models will be released - create & pass a conditions provider
```

Comment:
maybe already add this; we can release the mouse model now cc @AlexEMG @n-poulsen ?